### PR TITLE
fix two deep field renames at the same time

### PR DESCRIPTION
**Pull request by andimarek**

### CODE REVIEW

File: src/main/java/graphql/nadel/engine/FieldIdUtil.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    public static List<String> getFieldIds(Field field) {
+        String serialized = field.getAdditionalData().get(NADEL_FIELD_ID);
+        if (serialized == null) {
+            return Collections.emptyList();
+        }
+        List<NadelInfo> nadelInfos = readNadelInfos(serialized);
+        return nadelInfos.stream()
+                .map(nadelInfo -> nadelInfo.id)
+                .collect(Collectors.toList());
+    }
```
Comment (bbakerman) [inline src/main/java/graphql/nadel/engine/FieldIdUtil.java]: This is almost the same code twice.  Can we make a helpere method

File: src/main/java/graphql/nadel/engine/FieldIdUtil.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+        NadelInfo newNadelInfo = new NadelInfo(id, rootOfTransformation);
+        nadelInfos.add(newNadelInfo);
+        String newSerializedValue = writeNadelInfos(nadelInfos);
+        builder.additionalData(NADEL_FIELD_ID, newSerializedValue);
+
+    }
+
+    private static String writeNadelInfos(List<NadelInfo> nadelInfos) {
+        try {
```
Comment (bbakerman) [inline src/main/java/graphql/nadel/engine/FieldIdUtil.java]: try with resources rto auto close?

File: src/main/java/graphql/nadel/engine/FieldIdUtil.java (hunk 1 above)
Comment (andimarek) [reply src/main/java/graphql/nadel/engine/FieldIdUtil.java]: done

File: src/main/java/graphql/nadel/engine/FieldIdUtil.java (hunk 2 above)
Comment (andimarek) [reply src/main/java/graphql/nadel/engine/FieldIdUtil.java]: add try resources

**Pull request merged**